sdk/core/azure-mgmt-core/azure/mgmt/core/policies/_authentication.py

```diff
@@ -46,6 +48,11 @@ class ARMChallengeAuthenticationPolicy(BearerTokenCredentialPolicy):
     :param str scopes: required authentication scopes
     """
 
+    def __init__(self, credential: "TokenCredential", *scopes: str, **kwargs: Any) -> None:
+        # ARM supports Continuous Access Evaluation (CAE). This policy will enable it by default.
+        kwargs.setdefault("enable_cae", True)
```

Comment:
@pvaneck  Shall we wait for azure-core==1.28.1 release then update [setup.py of azure-mgmt-core](https://github.com/Azure/azure-sdk-for-python/blob/87703597ef919062235ce7c50bd5ab4ebc5186bf/sdk/core/azure-mgmt-core/setup.py#L71). After that, merge the PR and release a new version for azure-mgmt-core ?

Reply:
Hey, I wouldn't be opposed to it, @xiangyan99, what do you think? You think this would warrant a minimum core version bump for `azure-mgmt-core`? I suppose we are technically setting a keyword arg that will only be handled in the latest core. Although behavior changes with CAE being enabled/disabled mainly depends on if the latest identity version is used or not.

Reply:
Waiting for azure-core 1.29.0 is a good idea.

My question is do we want to make enable_cae default to True?

Are you know if all mgmt libraries handle challenge correctly?

Client libraries decided to set enable_cae back to False by default because if a library does not handle challenge correctly, it crashes. Hence we decided to make CAE an opt-in functionality and needed each library to enable explicitly.